### DISCUSSION TITLE
Why does the image not pop up when you add an image and tabs?

### DISCUSSION THREAD

**Discussion by Glacier956** [lepture/shibuya#64] [Q&A] [answered]
**Steps to repeat:**
1. 1. Add an image to the .rst file via thumbnail
.. thumbnail:: /_images/test.png
    :group: example-images
    :title: Test
2. Add tabs
`.. tabs::
	.. tab:: Test
            Test`

**The actual result:**
When you click on the image, it opens not with a pop-up, but as in a new tab

**Additional information**
If you remove tabs, the image opens with a pop-up as it should

Is this a bug?

**Comment by lepture** [comment, marked as answer]
The `tabs` directive is provided by `sphinx-design` extension. I think you need to report it to `sphinx-design`.
